### DISCUSSION TITLE
How to convert links automatically?

### DISCUSSION THREAD

**Discussion by poteirard** [remarkjs/remark#560] [Q&A] [answered]
I'm working on an issue for [react-styleguidist](https://github.com/styleguidist/react-styleguidist/pull/1710) and I realized that after upgrading Remark from 11 to 13 the links are no longer autogenerated.

For example:

Input: `http://example.com`

Output in 11: `[http://example.com](http://example.com \"http&#x3A;//example.com\")`
Output in 13:  `http://example.com`

How can I achieve automatic links?

Thank you!

**Comment by ChristianMurphy** [comment, marked as answer]
automatic links are a part of the Github Flavored Markdown standard (GFM) and can be added with https://github.com/remarkjs/remark-gfm

**Reply by poteirard** [reply]
Thanks a lot @ChristianMurphy
